javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -122,8 +122,11 @@ public static boolean areSameByName(
         if (a1 == a2) {
             return true;
         }
-        if (a1 == null || a2 == null) {
-            return false;
+        if (a1 == null) {
+            throw new BugInCF("Unexpected null first argument to areSameByName");
```

Comment:
The parameters are still annotated with `@Nullable`.

Reply:
Thanks for noticing! I adapted the two method signatures.